src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -116,6 +116,11 @@ INotationInteractionPtr NotationViewInputController::viewInteraction() const
     return m_view->notationInteraction();
 }
 
+Element* NotationViewInputController::hitElement() const
+{
+    return m_view->notationInteraction()->selection()->hitElementContext().element;
```

Comment:
Maybe better move `hitElementContext` to `notationInteraction` (not `selection`), because may be hit, but not "selection"

Reply:
moved